platform/api/packet/api.go

```diff
@@ -249,6 +249,17 @@ func (a *API) CreateDevice(hostname string, conf *conf.Conf, console Console) (*
 		return nil, fmt.Errorf("timed out waiting for flatcar-install: %v", err)
 	}
 
+	if id != "" {
+		// TCP discard service has been reached so `flatcar-install` is done.
+		// We can deactivate `PXE` boot to avoid bootlooping.
+		alwaysPXE := false
+		if _, _, err = a.c.Devices.Update(id, &packngo.DeviceUpdateRequest{
+			AlwaysPXE: &alwaysPXE,
+		}); err != nil {
+			plog.Errorf("unable to deactivate PXE boot: %v", err)
```

Comment:
```suggestion
			return nil, fmt.Errorf("unable to deactivate PXE boot: %v", err)
```
With PXE boot enabled it makes more sense to stop here or we might get strange errors later